qmarkdowntextedit.cpp

```diff
@@ -474,6 +476,13 @@ void QMarkdownTextEdit::centerTheCursor() {
     */
 }
 
+void QMarkdownTextEdit::signalNewLinePosition()
+{
+    auto cursor = textCursor();
+    cursor.movePosition(QTextCursor::StartOfLine);
+    emit newLinePosition(cursor.position());
```

Comment:
This should either be StartOfBlock or just cursor.block.position()

And since it's emitting for every cursor change, maybe this function isn't needed. The listener can directly listen to cursorPositionChanged and decide what to do.